bin/publish_workflow_json.sh

```diff
@@ -0,0 +1,34 @@
+PR_TITLE="chore: publish workflow-ui files"
+BRANCH_NAME="publish_workflow_ui_$(date +%s)"
+
+git switch -c $BRANCH_NAME
+
+# install jq
+sudo apt-get install jq
+
+# Loop through each package, install from npm and copy the workflow-ui.json from root of the package if it exists
+cat site/plugins.json | jq ".[] | select(.workflow == true) | .package" | while read PACKAGE
+do
+  echo "Installing $PACKAGE"
+  # remove the quotes from the package name
+  PACKAGE=$(echo $PACKAGE | tr -d '"')
+  npm install $PACKAGE
```

Comment:
Is there a reason to install the package vs pulling the file directly from NPM?

We only need raw files, not to pull dependencies and such

Reply:
nice, will use https://unpkg.com

Reply:
example being
```
npm pack $PACKAGE
tar -xf *.tgz
```
now a package directory will be there

Reply:
Suppose we'll need the whole package for the dynamic scripts and such

This is what I was using currently but I'll use `npm pack` instead

```bash
curl -s https://unpkg.com/$PACKAGE@latest/workflow-ui.json -o site/$PACKAGE/workflow-ui.json || true
```